resolver/resolver.cc

```diff
@@ -108,6 +108,14 @@ class ResolveConstantsWalk {
 
         AncestorResolutionItem(const AncestorResolutionItem &rhs) = delete;
         const AncestorResolutionItem &operator=(const AncestorResolutionItem &rhs) = delete;
+
+        inline core::SymbolRef symbolForStubbing() const {
+            if (isSuperclass) {
+                return core::Symbols::StubSuperClass();
+            } else {
+                return core::Symbols::StubMixin();
+            }
+        }
```

Comment:
It's a ~little strange to see this implemented as an instance method instead of a private helper function (throughout the entire history of the resolver we've treated these structs as dumb bags of data).

What do you think about moving this to a private function defined inside ResolveConstantsWalk that accepts an `AncestorResolutionItem` as input? This is how ~every other helper function in ResolveConstantsWalk is implemented.

Reply:
oh whoops I see Dmitry already said this.